src/main/scala/io/sdkman/changelogs/java/SapMachineMigrations.scala

```diff
@@ -202,4 +202,56 @@ class SapMachineMigrations {
       )
     ).validate().insert()
   }
+  @ChangeSet(
+    order = "0010",
+    id = "0010-add_sapmchn_jdk_11.0.8",
+    author = "poad"
+  )
+  def migrate0010(implicit db: MongoDatabase) =
+    Map(
+      Linux64 -> ("11.0.8", "linux-x64_bin.tar.gz"),
```

Comment:
The version doesn't change, so shall we just hard code it in the URL from now on? Then we can keep this as a simple map of KVs.

Reply:
I see!

The version appears twice, so I made it logical, but it's certainly common as a URL, so it's more concise to hardcode it in one line.